modules/react/breadcrumbs/lib/BreadcrumbsCurrentItem.tsx

```diff
@@ -47,14 +65,12 @@ export const BreadcrumbsCurrentItem = createSubcomponent('li')({
       <OverflowTooltip {...tooltipProps}>
         <Text
           as={Element}
-          typeLevel="subtext.large"
-          fontWeight="medium"
-          display="inline-block"
-          maxWidth={maxWidth}
-          whiteSpace="nowrap"
-          textOverflow="ellipsis"
-          overflow="hidden"
-          {...elemProps}
+          {...handleCsProp(
+            elemProps,
+            breadcrumbsCurrentItemStencil({
+              maxWidth: typeof maxWidth === 'number' ? px2rem(maxWidth) : maxWidth,
+            })
+          )}
```

Comment:
isn't Text already use `cs` prop? Ii's possible that you don't need `handleCsProp` here.
```
as={Element}
{...elemProps}
cs={
   breadcrumbsCurrentItemStencil({
     maxWidth: typeof maxWidth === 'number' ? px2rem(maxWidth) : maxWidth,
   })
}
```

I'm not sure, by the way :3

Reply:
No worries! Yeah this is part of us defining our patterns of how we style things vs external teams.

Here's my thinking:
- If a Stencil gets applied to an element, ie `nav` then we use `mergStyles`
- If a Stencil gets applied to another component that handles the cs prop, ie `Text`, then we use `handleCsProp` for consistency. We don't have to guess at what level the `cs` prop is being handled.

HOWEVER:
I'm open to suggestions here because we. haven't fully defined how to do this so the pattern is evolving!

Reply:
If someone passes a `cs` prop to the `BreadcrumbCurrentItem` component, we're overriding it hear. That means the `cs` prop is no longer supported.

We should only use the `cs` prop in examples. The `handleCsProp` and `mergeStyles` style functions handle `cs` prop merging.

Reply:
```
Here's my thinking:
- If a Stencil gets applied to an element, ie `nav` then we use `mergStyles`
- If a Stencil gets applied to another component that handles the cs prop, ie `Text`, then we use `handleCsProp` for consistency. We don't have to guess at what level the `cs` prop is being handled.
```
This is incorrect, IMO.

1. `mergeProps` should ONLY be used for components with style prop support. And we should generally be phasing it out as style props are going to be deprecated. 
2. `handleCsProp` should be used on any component that exposes the `cs` prop (so internally on every component in Canvas Kit)
3. `cs` can be used externally by consumers, but not in Canvas Kit (for the reasons Nicholas outlined)
4. `createStyles` should be used for one-off styles on elements.